app/pom.xml

```diff
@@ -75,7 +75,7 @@
     <errorprone.version>2.3.1</errorprone.version>
 
     <!-- Global camel version used everywhere -->
-    <camel.version>2.21.0.fuse-000094</camel.version>
+    <camel.version>2.21.0.fuse-710018</camel.version>
```

Comment:
Hi @dhirajsb, is this correct camel version? The productized version is **2.21.0.fuse-710018-redhat-00001** as stated in this jira [comment](https://issues.jboss.org/browse/ENTESB-9336?focusedCommentId=13626027&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-13626027)

Reply:
That's the pinned version on Origin nexus, we can't use the redhat version upstream.

Reply:
Just wondering, will it be changed during the build process by PNC?

Reply:
Yes it will

Reply:
thank you

Reply:
@apodhrad that's a prod build pinned in maven central, but it's pretty close to the one you mentioned, which is only in RH repos.

Reply:
In any case, that version is overwritten/fixed in the prod build process.